salt/states/rabbitmq_vhost.py

```diff
@@ -71,10 +71,11 @@ def present(name):
     vhost_exists = __salt__['rabbitmq.vhost_exists'](name)
 
     if __opts__['test']:
-        ret['result'] = None
         if vhost_exists:
+            ret['result'] = True
```

Comment:
This is a little nit-picky, but since `result: True` is already set above, we don't need this line here.